examples/applications/plot_stock_market.py

```diff
@@ -164,6 +164,7 @@
 # Learn a graphical structure from the correlations
 edge_model = covariance.GraphicalLassoCV()
 
+# %%
```

Comment:
looks like there's a bunch of these that should not be added

Reply:
I added a few manually in https://github.com/scikit-learn/scikit-learn/pull/17068/commits/9a63b6ee981a756fdfd5cf61f6a170a71c387892 because I though they should have been cells in the first place. Can revert if you prefer.

Reply:
I would revert and have these adjustments be made separate PRs. 

With this PR, we can keep the same formatting as master.

Reply:
I would revert these and have other PRs insert them.